quick_select/QuickSelect.java

```diff
@@ -0,0 +1,59 @@
+public class QuickSelect {
+    /*
+    * partition function
+    * array        : array on which partitioning has to be done
+    * left      : left index of the partitioning subarray
+    * right     : right index of the partitioning subarray
+    * pivot_idx : pivot index from which partition has to done
+    * store_idx : index of the last element of the left subarray
+    * return    : the index of the last element of the left subarray
+     */
+    public static int partition(int[] array, int left, int right, int pivot_idx) {
```

Comment:
This method should be `private`. Also variables should follow `camelCase`. Please change it.